pkg/oauth/user.go

```diff
@@ -207,16 +211,49 @@ func (s *server) Login(c echo.Context) error {
 	return c.NoContent(http.StatusNoContent)
 }
 
+var errInvalidLogoutRedirectURI = errors.DefinePermissionDenied("invalid_logout_redirect_uri", "the redirect URI did not match the one(s) defined in the client")
+var errMissingAccessTokenIDParam = errors.DefinePermissionDenied("missing_param_access_token_id", "Access token ID was not provided")
```

Comment:
1. Formatting: you can also put these in a `var` block, and use some newlines to make it more readable:
	```go
	var (
        errInvalidLogoutRedirectURI = errors.DefinePermissionDenied(
			"invalid_logout_redirect_uri", 
			"the redirect URI did not match the one(s) defined in the client",
		)
		errMissingAccessTokenIDParam = // ...
	)
	```
2. Lowercase "Access" in description of the second